py/desitarget/streams/gaia_dr3_parallax_zero_point/zpt.py

```diff
@@ -0,0 +1,292 @@
+# created by: Pau Ramos (@brugalada) & Anthony Brown
+# based on Matlab code by: Lennart Lindegren
+# Date: 21/07/2020  (important for the table of coefficients)
+
+import os
+import numpy as np
+import warnings
+
+__mypath = os.path.dirname(os.path.abspath(__file__))
+
+_file5_currentversion = __mypath + '/coefficients/z5_200720.txt'
+_file6_currentversion = __mypath + '/coefficients/z6_200720.txt'
```

Comment:
Although this is the method used by the current code, I'm not sure it will survive the pip installation process.  Let's update this to use `importlib.resources.files` (which replaces deprecated `pkg_resources`).

I think you also need to add the coefficients files to setup.py `setup_keywords['package_data']` so that they will actually be installed.  Ping me on Slack for installation testing.  FTR: the setup.py installation method itself is also deprecated, but that should refactored in a separate PR from this one.

Reply:
I've updated the code to use `importlib.resources.files`. I'll test the `setup_keywords` install next.

Reply:
I think I've now also correctly updated setup.py but I'll check with Stephen.

Reply:
Stephen and I checked the installation.